cli/command/node/client_test.go

```diff
@@ -0,0 +1,85 @@
+package node
+
+import (
+	"io"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/swarm"
+	"github.com/docker/docker/cli/command"
+	"github.com/docker/docker/client"
+	"golang.org/x/net/context"
+)
+
+type fakeCli struct {
+	command.DockerCli
+	client client.APIClient
+	out    io.Writer
+}
```

Comment:
Same as above, I definitely think it could be useful for othe tests. It felt simpler and easier to use a fake than use another dependency. If it gets too complicated I would be ok to use it and/or generate it 👼 (I tend to prefer fake to mock when it's simpler to write — but yeah should take a closer look at `mockery` in the near future).